web/iot/simulation.js

```diff
@@ -113,7 +113,8 @@ async function initializeSimulators () {
 
 async function createSimulator (args) {
   return new Promise((resolve, reject) => {
-    var simulatorTask = new Worker(URL.createObjectURL(new Blob(["("+worker_node.toString()+")()"], {type: 'text/javascript'})));;
```

Comment:
Can this be made dynamic?  To use one technique when running from the 'file' protocol, else use the other technique when running from a server?

Reply:
I need to test this but if we have some configuration file on GitHub Web Pages we should be able to do that.